packages/gigamon/changelog.yml

```
@@ -0,0 +1,6 @@
+# newer versions go on top
+- version: "0.1.0"
+  changes:
+    - description: Initial draft of the package
```

```suggestion
    - description: Initial release
```

`draft` doesn't seem to be the right word.